designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -593,13 +603,25 @@ class DeploymentServiceImpl(
           params
         )
         customActionOpt = manager.customActionsDefinitions.find(_.name == actionName)
-        _ <- existsOrFail(customActionOpt, CustomActionNonExisting(actionName))
+        _ <- existsOrFail(customActionOpt, CustomActionNonExistingError(s"${actionName.value} doesn't exist"))
+        validator = new CustomActionValidator(manager.customActionsDefinitions)
+        // TODO: remove this validation prosthesis
```

Comment:
Please remove this validation prosthesis and use action parameters validation.

Reply:
Used Validator